registrants/src/UI/embc-registrant/src/app/feature-components/eligible-self-serve/eligible-self-serve-support-form.component.html

```diff
@@ -1,8 +1,18 @@
 @if (draftSupportError) {
-  <div style="display: block; color: red; background-color: orange; padding: var(--size-2)">
+  <div style="display: block; color: white; background-color: orange; padding: var(--size-2); font-size: var(--size-3)">
     Error: Unable to retrieve Draft Support information.
   </div>
 }
+@if (submitSupportError) {
+  <div style="display: block; color: white; background-color: orange; padding: var(--size-2); font-size: var(--size-3)">
+    Error: Unable to submit Support information.
+  </div>
+}
+@if (calculateTotalsError) {
+  <div style="display: block; color: white; background-color: orange; padding: var(--size-2); font-size: var(--size-3)">
```

Comment:
shouldn't this be moved to a css class? it's being repeated multiple times

Reply:
Yes, will fix that.